fahclient-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+# assumes https://github.com/NVIDIA/k8s-device-plugin or similar is deployed
```

Comment:
I doubt people will be running this container on kubernetes, but it doesn't hurt to have an example.

Reply:
Actually I'm currently writing a helm chart which people can use to install FAH on a Kubernetes cluster. :)

Reply:
I just released a helm chart which can be used to install on a Kubernetes cluster: https://github.com/pcktdmp/fahclient

Reply:
I am running this container on kubernetes.

Reply:
Would this also work in a simple docker container? I'm running FAH on my personal machine, and would greatly prefer avoiding having to deploy k8s on it (tried that before, and wasted about 3 days of work with tons of hypervisor VMs only for the thing to total fail).

Reply:
It's definitely not need Christopher, if you look at the README.md in the root of this repo you can just simply use a single docker container with the `docker` command. :-)
This file is just added for /if/ people want to run it on Kubernetes.